System.Containers.Tasks/ParseContainerProperties.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Resources;
+using System.Text;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System.Containers;
+
+#nullable disable
```

Comment:
For production code I'd like to leave this on if we can

Reply:
had to default all strings in the constructor to `""`, which I wasn't too happy about, but I didn't want to mark all strings (which msbuild handles anyway) as nullables.